### PR TITLE
Initial version of explainer tutorial with samples

### PR DESCRIPTION
**Pull request by cromwellian**

### CODE REVIEW

File: pkg/demo/explainer/CustomFontSet.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+export const fonts = [
```
Comment (shaper) [inline pkg/demo/explainer/CustomFontSet.js:1]: Needs header.

File: pkg/demo/explainer/Library/1-NoOpDecorator.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC.
```
Comment (shaper) [inline pkg/demo/explainer/Library/1-NoOpDecorator.js:3]: 2022 (and same for other new files).

File: pkg/demo/explainer/Library/2-SimpleDecorator.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+      fonts: {
+        models: fonts,
+        $template: 'font_t',
+        decorator: 'decorator'
+      }
+    };
+  },
+
+  decorator({family, fullName, weight, style}) {
+    const fweight = style.includes('Bold') ? 'bold' : weight;
```
Comment (shaper) [inline pkg/demo/explainer/Library/2-SimpleDecorator.js:23]: This helper logic is common, maybe worth moving to some shared util at some point? I'm not sure what the right pattern is for that in this Arcs impl.

File: pkg/demo/explainer/fulldemo.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=0.86, maximum-scale=3.0, minimum-scale=0.86, viewport-fit=cover">
+<link rel="stylesheet" href="style.css">
+
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Bebas+Neue&family=Comforter+Brush&display=swap');
+  #test {
+    font-family: sans-serif;
+    /*
+    font-family: 'Architects Daughter';
```
Comment (shaper) [inline pkg/demo/explainer/fulldemo.html:12]: Do we need these?

File: pkg/demo/explainer/index.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+<html lang="en" style="opacity: 0;">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=0.86, maximum-scale=3.0, minimum-scale=0.86, viewport-fit=cover">
+<link rel="stylesheet" href="style.css">
+
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Bebas+Neue&family=Comforter+Brush&display=swap');
+  #test {
+    font-family: sans-serif;
+    /*
```
Comment (shaper) [inline pkg/demo/explainer/index.html:11]: Do we need these?

**Review by shaper — APPROVED** [review]

File: pkg/demo/explainer/Library/2-SimpleDecorator.js (hunk 3 above)
Comment (cromwellian) [reply pkg/demo/explainer/Library/2-SimpleDecorator.js:23]: I don't think you can 'import' in non-permissive particles? Maybe Scott can provide a helper function for dynamic imports, but I think you'd need to "await" it.

Comment (sjmiles) [reply pkg/demo/explainer/Library/2-SimpleDecorator.js:23]: With`Runtime.particleOptions` you can inject whatever API you like into Particle-scope:
```
// customize particle scope
Runtime.particleOptions = {
  injections: {
    // items to make available in each Particle's module scope
    fetch: fetch.bind(window)
  }
};
```

**Pull request merged**